### DISCUSSION TITLE
Image is rotated for jpeg to pdf rendition

### DISCUSSION THREAD

**Discussion by ptiruvaipati** [ImageMagick/ImageMagick#7536] [Help]
### ImageMagick version

7.1.1-33

### Operating system, version and so on

Windows 10

### Description

Hi,

We are using ImageMagick 7.1.1-33.

Tried to transform jpeg to pdf , the image is rotated 180 degree in the pdf output file. 
Source file, output files are attached. 

[Input File](https://github.com/user-attachments/assets/ba578221-1464-46c5-85ac-56977eef5cfe)

[Output.pdf](https://github.com/user-attachments/files/16522634/Output.pdf)

Can you please check this issue?

Regards,
Pavan

**Comment by snibgo** [comment]
Your input image has orientation metadata. IM ignores this, unless you use `-auto-orient` to correct the orientation. For example:
```
magick in.jpg -auto-orient out.pdf
```
